custom/ewsghana/reports/__init__.py

```diff
@@ -61,6 +61,8 @@ class MultiReport(CustomProjectReport, CommtrackReportMixin, ProjectReportParame
     report_template_path = "ewsghana/multi_report.html"
     flush_layout = True
     split = True
+    exportable = True
+    is_exportable = False
```

Comment:
@czue Every report in EWS should be exportable because when we select facility in location filter then Facility Page Report will be loaded. I also needed one additional flag to define if report in the other cases is exportable or not and if not I just hide export button.
I needed 3 states of this flag and thats why I added one more (also one of them should change dynamically according to what we select in location filter).